drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -162,7 +162,7 @@ def _resolve(input:, record_lock_version: nil)
       when 'Hmis::Hud::Funder', 'Hmis::Hud::ProjectCoc', 'Hmis::Hud::Inventory', 'Hmis::Hud::CeParticipation', 'Hmis::Hud::HmisParticipation'
         [project, klass.new({ project_id: project&.project_id, **ds })]
       when 'Hmis::Hud::Enrollment'
-        [project, klass.new({ project_id: project&.project_id, personal_id: client&.personal_id, **ds })]
+        [project, klass.new({ project_pk: project&.id, personal_id: client&.personal_id, **ds })]
```

Comment:
only assigning the project_pk will create a WIP enrollment initially. That's probably what we want?

Reply:
That logic of whether it should be saved as WIP or non-WIP is handled by this method:
https://github.com/greenriver/hmis-warehouse/blob/85d50fadd0639c95d25824bdd5657ddc32782761/drivers/hmis/app/models/hmis/hud/enrollment.rb#L291-L302

So I think here you can set both ProjectID and project_id, and `ProjectID` will get nulled out by  `save_in_progress!` if its called.